src/config/eww_config.rs

```diff
@@ -50,7 +76,7 @@ impl EwwConfig {
                 Ok((def.name.to_owned(), def))
             })
             .collect::<Result<HashMap<_, _>>>()
-            .context("error parsing window definitions")?;
+            .context(format!("error parsing window definitions: {}", path.display()))?;
```

Comment:
```suggestion
            .with_context(|| format!("error parsing window definitions: {}", path.display()))?;
```
with_context makes this more performant, as it only does the string formatting computation when the erorr actually happens.